libs/vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -148,7 +149,8 @@
     "response_logprobs",
     "logprobs",
 ]
-_allowed_params_prediction_service = ["request", "timeout", "metadata"]
+_allowed_params = [*_allowed_params, "labels"]
```

Comment:
nits: can you just modify the original _allowed_params, please? why do we need a re-assignment of the variable?